### PR TITLE
Implement component CartItem

### PR DESCRIPTION
**Pull request by thanhtrantruong47**
Implement component CartItem

image

image

storybook: https://6684faab2d0c99a3e93bf145-okzryfskef.chromatic.com/

### CODE REVIEW

File: src/components/CartItem/CartItem.stories.tsx [hunk 1]
```diff
@@ -9,5 +9,13 @@ export default {
...
...sK6uRvR1GQWnWnmlCEsHODJDki44+NMa/mhcUTdxJsG7Ed6rPDhx59fe9eEuNhKLQkGcBZNZBCUQ60s8je0WoABwj7i/CYlTurl32vF2fhHhsOvXLA/h9WIPfpCuBg0ZJFiHvm6xWHnEC94rsa1MBEE7wOsPrOJnDDDKjpmOIEBzW5xf2/8cUEHXXXcqlmbvvKXm3JrTQwm7yfEuMM7cRxctVg2h9d0OXRDjEXLqVx3QTBzgycrANw8jS5jQt9foRx0vUwQquTWj3xuQViqjQbDS1B+JTRP7GNsPQeFIUcUS5/1J+3B2xUu3SM0MUuuu0McTIlQNUztbt+Gfh0B3G3Oe9zGf8paYvKucQmRIGCZTARjj/f1/UA6riP9Btgc4D9uFjuO3A32c3sIm4VHd4q2UbiH9BdaYcWqXhJ7JUhmI6/TuGv3jdIPkUyH+IvZlfJcJVX/ovW71NiyPUTkGxmh3+wAFMUqukK3z0zfl4qf/+vrDjBV3QnXexJLh7zH+/SPtmpk6dyPZ0UH8CJPwzL0qRDvYUweE/U9T5KPglNP/VXGjd2bIWqE6A7Y35rgSk+zoyd/17w/NmuvwFCK8dSyC402oHkjCznTpkr0THCTybDQ9hQYlz7r+ZTiRassMBOk7mlUS/1+9+ZGGU/xaoE5WyUJuMbXpTT1ATQ+fhFabvHirIIkte09q/aQJW/+hUgf70SUMt6/HdW7Ows+HEWDKYPWfiB0tykeJgae7HccXQdGZkTkEO/KCzqQc0s8GOfKM89s6MGSIYJ26K0qepo85JdNpD8mTXPanVx96Kj9eKDCrUEo96GYzrbqswXu6F6PVTIkGKrWAEOyGRuPqULP3WQHNkeF4f8yhTQ26Yr6BFS7Ia+qpPoYVYNi5s/QRa/Xz26B09ahuuZVhcqo/BHokb78A6TkISDh3mA9y30Czp87iAfXZ4QnqZA+PRzdBbEtvoIIXZTUUbDshekxNzrrVsMLquJAS0/0LEpgPZm4B1omU2DF1G40FH/cEmudhcGKOJ4WTUZ2DnwZmomb8fwvqU9HSE44ZYLypWMXz9MUZlMAIkO3RgccZbHQqAI9bBgkP4miGAR7AKbfm0YT9ILXh5i4bDBH/5/H0FV37KqwB96w0SGxHSn5NbMYGCJiA/lsGBtvEG7qjEKjTnZbbVImQYVrXf4g91rEAWH3KAAAAA=',
```
Comment (thanhnguyen-agilityio) [inline src/components/CartItem/CartItem.stories.tsx]: Use url image

File: src/components/CartItem/cartItem.module.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
+.container {
+  display: grid;
+  background-color: var(--color-bg);
+  padding: 2.5rem;
+  gap: 2.875rem; /* 46px */
+  max-width: 48.875rem; /* 782px */
```
Comment (thanhnguyen-agilityio) [inline src/components/CartItem/cartItem.module.css]: It should not have max width. The page container should has max-width and this should be x/y of container.

File: src/components/CartItem/CartItem.stories.tsx (hunk 1 above)
Comment (thanhtrantruong47) [reply src/components/CartItem/CartItem.stories.tsx]: Fixed

File: src/components/CartItem/cartItem.module.css (hunk 2 above)
Comment (thanhtrantruong47) [reply src/components/CartItem/cartItem.module.css]: Fixed

**Review by GraphicDThanh — APPROVED** [review]

**Review by thanhnguyen-agilityio — APPROVED** [review]

**Pull request merged**